### DISCUSSION TITLE
drop in replacements for GP to estimate infection timeseries

### DISCUSSION THREAD

**Discussion by smwindecker** [idem-lab/lowerGPreff#13] [Ideas]
We should consider a suite of underlying models for infection incidence timeseries, including more mechanistic ones, for future extensions of the software. 

drop in replacements for GP

**Comment by AugustHao** [comment]
in those implementations we might need a time-varying, flexible deviance term (eg a GP over time) to absorb the residual deviance between the mechanistic model output and the observed data, much like component 2 in the TP reff model

**Comment by smwindecker** [comment]
enable user to account for susceptible depletion in reff forecast.
